### PR TITLE
[Clients][Core & Internals]: Add TTC to rucio rule-info command Fix #530

### LINKED ISSUE #530: Functionality to include TTC for a rule in rucio rule-info
Motivation
----------
Rucio rule-info should display the TTC.

### PR DESCRIPTION
**Pull request by jwackito**
[Clients][Core & Internals]: Add TTC to rucio rule-info command Fix #530

### CODE REVIEW

File: bin/rucio [hunk 1]
```diff
@@ -2766,6 +2775,7 @@ Commands:
     info_rule_parser.set_defaults(which='info_rule')
     info_rule_parser.add_argument(dest='rule_id', action='store', help='The rule ID')
     info_rule_parser.add_argument('--examine', dest='examine', action='store_true', help='Detailed analysis of transfer errors')
+    info_rule_parser.add_argument('--include-ttc', dest='get_ttc', action='store_true', help='Show Estimated Time To Complete for the rule. Calculation can be time consuming.')
```
Comment (bari12) [inline bin/rucio]: Change `get_ttc` to `estimate_ttc`

Comment (jwackito) [reply bin/rucio]: You mean the name of the variable or the name of the argument? The command will be invoked as
rucio rule-info --include-ttc 12abc..1234abc
and you wanted to change it to 
rucio rule-info --get-ttc 12abc..1234abc
?

Comment (mlassnig) [reply bin/rucio]: --estimate-ttc

Comment (jwackito) [reply bin/rucio]: Done, yet I prefer --include-ttc :P

**Comment by bari12** [comment]
I think it is fine now; Just one minor comment, you are using three different names/variables for the same thing; estimate-ttc, get_ttc, include_ttc - It would improve readability to reduce this to one.

**Comment by jwackito** [comment]
Done. All variables move to estimate_ttc :)

**Review by bari12 — APPROVED** [review]

**Pull request merged**